<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settng_TrainCode</class>
 <widget class="QWidget" name="Settng_TrainCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:black;
}
QPushButton
{
background-color:black;
color:white;
border:1px solid white;
font:14px;
}</string>
  </property>
  <widget class="QPushButton" name="Button_CurrentCarNum">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>本车车号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_OtherCarNum">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>它车车号</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CurrentCarNum">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>120</y>
     <width>131</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_OtherCarNum">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>120</y>
     <width>131</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num1">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_A">
   <property name="geometry">
    <rect>
     <x>493</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num4">
   <property name="geometry">
    <rect>
     <x>553</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num6">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>280</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num7">
   <property name="geometry">
    <rect>
     <x>493</x>
     <y>280</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num8">
   <property name="geometry">
    <rect>
     <x>553</x>
     <y>280</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num2">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num3">
   <property name="geometry">
    <rect>
     <x>493</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num9">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_B">
   <property name="geometry">
    <rect>
     <x>553</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num0">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num5">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>280</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Tips">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:14px;</string>
   </property>
   <property name="text">
    <string>请输入车号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_InputCarNum">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_SetCurrentCar">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>310</y>
     <width>99</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>本车设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_SetOtherCar">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>354</y>
     <width>99</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>它车设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Clear">
   <property name="geometry">
    <rect>
     <x>493</x>
     <y>400</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
